plugins/cilium-cni/cilium-cni.go

```diff
@@ -262,7 +268,18 @@ func setupLogging(n *types.NetConf) error {
 	logOptions := logging.LogOptions{
 		logging.FormatOpt: f,
 	}
-	return logging.SetupLogging([]string{}, logOptions, "cilium-cni", n.EnableDebug)
+	err := logging.SetupLogging([]string{}, logOptions, "cilium-cni", n.EnableDebug)
```

Comment:
The current signature of SetupLogging function is taking slice of strings for logger, which makes it hard to configure new kind of loggers with different parameters such as file rotation. Considered that this function is widely used in codebase, I just go with log hook approach in this PR. Feel free to let me know if you have better idea.